core/processor/src/main/java/io/quarkus/annotation/processor/generate_doc/ConfigDocItemFinder.java

```diff
@@ -494,7 +494,19 @@ private List<String> extractEnumValues(TypeMirror realTypeMirror, boolean useHyp
                 }
                 if (rawJavaDoc != null && !rawJavaDoc.isBlank()) {
                     // Show enum constant description as a Tooltip
-                    String javaDoc = enumJavaDocParser.parseConfigDescription(rawJavaDoc);
+
+                    // tooltip description is inline-block and renders incorrectly when it contains a paragraph
+                    final String javaDoc;
+                    if (rawJavaDoc.contains("<p>")) {
+                        String stripped = rawJavaDoc.replaceAll("<p>", Constants.EMPTY);
+                        if (stripped.contains("</p>")) {
+                            stripped = stripped.replaceAll("</p>", Constants.EMPTY);
+                        }
+                        javaDoc = enumJavaDocParser.parseConfigDescription(stripped);
+                    } else {
+                        javaDoc = enumJavaDocParser.parseConfigDescription(rawJavaDoc);
+                    }
+
                     acceptedValues.add(String.format(Constants.TOOLTIP, enumValue, javaDoc));
```

Comment:
I don't think it's worth overoptimizing things in this case. And also, I wouldn't use `replaceAll`.

```suggestion
                    acceptedValues.add(String.format(Constants.TOOLTIP, enumValue, javaDoc.replace("<p>", Constants.EMPTY).replace("</p>", Constants.EMPTY));
```
should do it?

Reply:
> I don't think it's worth overoptimizing things in this case.

I'd say in 99 % there will be no replacing (and for example `SchedulerRuntimeConfig` does not contain `</p>`). Judging by implementation I use, the `contains` method should be better than `replace` because it doesn't recreate String.

> And also, I wouldn't use replaceAll.

I agree, `replaceAll` was my mistake.

Reply:
I forgot to say I'll commit your suggestion (or you can do it).